units/projectile.tscn

```diff
@@ -8,6 +8,7 @@ radius = 4.0
 
 [node name="Projectile" type="Area2D"]
 position = Vector2(68, 1)
+collision_layer = 2
```

Comment:
🤔 I kinda like the idea that projectiles can hit themselves / the player... but that'll take some thinking. For now 👍 👍

Reply:
Not a terrible idea, though the huge double lasers overlapped hitboxes, so they would just immediately disappear